pkgs/by-name/op/opentrack/aruco.nix

```
@@ -4,38 +4,39 @@
   cmake,
   fetchFromGitHub,
   opencv4,
+  opentrack,
 }:
 
 stdenv.mkDerivation {
   pname = "opentrack-aruco";
-  version = "unstable-20190303";
+  version = "unstable-2022-03-10";
 
   src = fetchFromGitHub {
     owner = "opentrack";
     repo = "aruco";
-    rev = "12dc60efd61149227bd05c805208d9bcce308f6d";
-    sha256 = "0gkrixgfbpg8pls4qqilphbz4935mg5z4p18a0vv6kclmfccw9ad";
+    rev = "e08d336bd2c70859efd19622582817fe8eabe714";
+    sha256 = "sha256-Wf5y//vxRWIZfOxwgZsPPTsUHliA3cbt2FOjouh4/rQ=";
```

You might want to change here for hash.
```suggestion
    hash = "sha256-Wf5y//vxRWIZfOxwgZsPPTsUHliA3cbt2FOjouh4/rQ=";
```